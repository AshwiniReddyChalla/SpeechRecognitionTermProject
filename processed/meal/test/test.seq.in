what meals are served on american flight DIGITDIGITDIGIT from tampa to milwaukee
what meals are served on american flight DIGITDIGITDIGIT DIGITDIGITDIGIT from milwaukee to seattle
what meals are available on dl DIGITDIGITDIGIT which al arrives in san francisco at DIGITDIGITDIGIT am
what meals are there on flight DIGITDIGITDIGIT from milwaukee to washington dc on tuesday morning
are meals ever served on tower air
are snacks served on tower air
