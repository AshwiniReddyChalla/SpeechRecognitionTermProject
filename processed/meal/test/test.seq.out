O B-meal O O O B-airline_name O B-flight_number O B-fromloc.city_name O B-toloc.city_name
O B-meal O O O B-airline_name O B-flight_number I-flight_number O B-fromloc.city_name O B-toloc.city_name
O B-meal O O O B-airline_code B-flight_number O O O O B-toloc.city_name I-toloc.city_name O B-arrive_time.time I-arrive_time.time
O B-meal O O O O B-flight_number O B-fromloc.city_name O B-toloc.city_name B-toloc.state_code O B-depart_date.day_name B-depart_time.period_of_day
O B-meal O O O B-airline_name I-airline_name
O B-meal_description O O B-airline_name I-airline_name
